backend/src/ast/converters/python/operators.ts

```diff
@@ -268,14 +268,20 @@ export const createSetOperator = "create-set";
 
 /**
  * Matches a set comprehension expression
- *     { x * 2 for x in range(10) if x % 2 == 0 }
- *     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+ * { x * 2 for x in range(10) if x % 2 == 0 }
+ * ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  */
 export const setComprehensionOperator = "set-comprehension";
 
 /**
  * Matches an additive inverse expression
- *   result = -value
- *            ^^^^^^
+ * result = -value
+ *          ^^^^^^
  */
 export const additiveInverseOperator = "additive-inverse";
+
+/**
+ * Matches a start (unpack) operator
+ * func(*args)
```

Comment:
wouldn't a better example here be the following?

```suggestion
 * a, b, c = *args
 *           ^^^^^
```